org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/staging/AbapGitStagingView.java

```diff
@@ -1273,7 +1309,7 @@ private void addDoubleClickListener(TreeViewer viewer) {
 			if (event.getSelection() instanceof IStructuredSelection) {
 				IStructuredSelection selection = (IStructuredSelection) event.getSelection();
 				Object object = selection.getFirstElement();
-				this.stagingUtil.openEditor(object, this.project, this.repositoryCredentials);
+				this.stagingUtil.openEditor(object, this.project);
```

Comment:
openEditor action, opens a local file/object in an editor. Hence does not require any credentials.